style.css

```diff
@@ -39,12 +39,63 @@ a.logo {
   }
 }
 
-a.union {
+.burger {
+  display: block;
```

Comment:
Good job for adding the Hamburger Icon. The hamburger icon is too small. Please increase the size a little bit. 👍 

Also, Please Increase the size of the Cross button as well. 👍

Reply:
Thank you. I will solve it right away.